x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/notification/slack/message/SlackMessage.java

```diff
@@ -29,6 +29,17 @@
     final String text;
     final Attachment[] attachments;
 
+    //Note on constructors:
+    // Slack with fail to send a message if it has no attachments AND no text.
+    // Slack will succeed with attachments OR text.
+    public SlackMessage(String from, String[] to, String icon, Attachment[] attachments) {
+        this(from, to, icon, null, null); // Empty text
```

Comment:
Shouldn't `attachments` be used here.

Reply:
++, I think this is a copy-paste from the last commit just with that missing.